### PR TITLE
As a user, I need to know the projection for a Shapefile export

### PR DESCRIPTION
**Pull request by ahervy**
## Summary

This pull request adds info about the default geographical projection for dataset exports. This pull request also includes some content restructuring for better readability.

Story details: https://app.clubhouse.io/opendatasoft/story/29638

## Changes 

- Added info about the [default projection and additional projections for geo file formats](https://github.com/opendatasoft/ods-documentation/commit/e7e58688ebe5f671658825a4bff40f50825b5bf5#diff-74c8168259e1d861f5b74f633846ba0ef565ae8895b4d9ce558881838e28dca4R52-R55)
- Moved available export formats to a [dedicated section](https://github.com/opendatasoft/ods-documentation/commit/e7e58688ebe5f671658825a4bff40f50825b5bf5#diff-74c8168259e1d861f5b74f633846ba0ef565ae8895b4d9ce558881838e28dca4R28-R72)
- Made all UI terms bold when users need to interact with them in instructions
- Moved the [notes about exporting data using the Search API v2](https://github.com/opendatasoft/ods-documentation/compare/feature/ch29638/projection-for-shapefile-exports?expand=1&title=As%20a%20user,%20I%20need%20to%20know%20the%20projection%20for%20a%20Shapefile%20export&body=Story%20details:%20https://app.clubhouse.io/opendatasoft/story/29638#diff-74c8168259e1d861f5b74f633846ba0ef565ae8895b4d9ce558881838e28dca4R18-R22) at the end of the procedure because the procedure was lost in the middle of the section
- Integrated the [note about exporting filtered data](https://github.com/opendatasoft/ods-documentation/commit/accd7b68b0a1c112873339200675b07be14a5bf4#diff-74c8168259e1d861f5b74f633846ba0ef565ae8895b4d9ce558881838e28dca4L16-R16) in the procedure to avoid multiple notes and improve content readability
- Improved the wording in intros and instructions

### CODE REVIEW

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst [hunk 1]
```diff
@@ -4,44 +4,68 @@ Exporting data
...
+   :header-rows: 1
+   :widths: 25 75
+
+   * * Format
+     * Notes
+   * * CSV
+     * CSV uses semicolon (;) as a separator. 
+   * * JSON
+     * 
+   * * Excel
```
Comment (cblsr) [inline source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst:42]: Maybe something to add with regards to our Excel output is the type of Excel, I know it has led to some problems in the past (due to Excel 2003 vs. 2007 vs. 2010…). In the ticket https://secure.helpscout.net/conversation/1443040226/18380/ , @NTerpo explained that it was in fact "a Microsoft Office XML 2003 with an XLS extension"… I don’t know if we want to publicly document it, though

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst [hunk 2]
```diff
@@ -4,44 +4,68 @@ Exporting data
...
+   * * Format
+     * Notes
+   * * GeoJSON
+     * 
+   * * Shapefile
+     * There are 2 limitations in Shapefile exports:
 
-   You can use the parameter :code:`use_labels_for_header=true` in the exports links to use the labels instead of the technical identifiers of the fields as a header (not available in API v2).
+        - The export cannot contain more than 50,000 records.
+        - The export cannot contain more than one geographical shape type (point, polygon, linestring, etc.). Only the first type of the first non-empty record will be kept in the export.
```
Comment (cblsr) [inline source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst:65]: When I read this phrase, I think of two different cases:
- a dataset with a "geoshape" field, in this field different types (point, polygon…), only the shapes of the same kind that the first shape found are kept in the export (but what about the data attached to the other shapes? are the records kept without their shape? or not present at all?)
- a dataset with multiple "geo" field (geoshape or geo_point_2d): I tried with https://public.opendatasoft.com/explore/dataset/georef-united-states-of-america-place-millesime/information/ : in the Information > Data model tab, there are both fields available, and geo_point_2d comes first; when exporting as Shapefile and looking at the export with QGIS, only the shape is present

I suppose that you were thinking about the first case, but maybe that the second one should be explained too?

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst [hunk 3]
```diff
@@ -4,44 +4,68 @@ Exporting data
...
+
+Geographic file formats
+~~~~~~~~~~~~~~~~~~~~~~~
 
 .. admonition:: Important
-   :class: important
+   :class: Important
 
-   If the dataset is filtered, it is possible to only download the filtered data. To do so, click on "Only the selected records" instead of "Whole dataset".
+   The geographical coordinates of a dataset are expressed in WGS84 by default. When available, you can select additional geographical projections from the drop-down displayed above the export formats.
```
Comment (cblsr) [inline source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst:51]: On the "when available", I would add something like "when made available by the data producer" to show the difference between "geo formats are available automatically when geographical data is present" and "geo projections are manually chosen by the data producer" (and the re-users could ask the producers about their preferred projection if it’s really important).

**Review by cblsr — APPROVED** [review]
Looks good to me, I left a few comments but nothing major!

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst (hunk 2 above)
Comment (ahervy) [reply source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst]: Fixed in 9da1777ab424022b02d96e9ff6512efdd7579538

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst (hunk 3 above)
Comment (ahervy) [reply source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst]: Fixed in 9da1777ab424022b02d96e9ff6512efdd7579538

File: source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst (hunk 1 above)
Comment (NTerpo) [reply source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst:42]: Imho we don't want to publicly document it for now. Let's wait for the work on the download / exports to see if that change anything on this topic and re-discuss it at that moment.

**Pull request merged**